src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -440,6 +440,27 @@ class AresDnsResolverFactory : public ResolverFactory {
 
 }  // namespace grpc_core
 
+extern grpc_address_resolver_vtable* grpc_resolve_address_impl;
+static grpc_address_resolver_vtable* default_resolver;
+
+static void resolve_address_ares(const char* addr, const char* default_port,
```

Comment:
Why bother with this function, since its signature seems to be the same as `grpc_resolve_address_ares()`?  Why not simply use `grpc_resolve_address_ares()` directly in the vtable?

Reply:
Fixed.